source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,119 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+While designing your database, you may want to enforce particular datatypes in specific fields. MongoDB uses JSON Schema to reduce the complexity of data validation during insert and update operations. Field Level Encryption with MongoDB requires the use of JSON Schema to specify several configuration options, allowing the drivers to encrypt and decrypt fields automatically.
```

Comment:
I know what I suggested is very similar to what you have written, but it still seems focused on JSON schema for data validation and not enough on automatic encryption/decryption. The last sentence is also focused on "several configuration options" rather than directly "encryption algorithm/scheme".

How about something like this:
```
MongoDB uses JSON Schema to enforce particular datatypes for specific fields during insert and update operations. MongoDB CSFLE requires JSON Schema to identify which fields to automatically encrypt and decrypt, the data encryption key id, and the type of encryption algorithm to use.
```

Reply:
Fixed to your paragraph roughly:

"MongoDB uses JSON Schema to enforce particular datatypes for specific fields during insert and update operations. MongoDB CSFLE requires JSON Schema to identify fields to automatically encrypt and decrypt, the data encryption key id, and the type of encryption algorithm to use."